_even.rb

```diff
@@ -5,3 +5,4 @@ def odd_or_even(num)
     puts 'odd'
   end
 end
+
```

Comment:
can you make it more specific to user friendly, so that user will understand after running the method what function will do.